Dockerfile

```diff
@@ -1,4 +1,26 @@
-FROM amd64/node:10.19.0@sha256:2903bbe55db122b227b20c445d1c06e2b1c44b6a0dcfab92734c006edd3d2b4f AS tsbuild
+ARG IMAGE=latest
+
+# Base image
+#============
+FROM renovate/yarn:1.22.0@sha256:22cfbb7ecd47655a905c01fe3f2fcb3ac34839dd6d42d81eeb437c80475483ea AS base
+
+LABEL maintainer="Rhys Arkins <rhys@arkins.net>"
+LABEL name="renovate"
+LABEL org.opencontainers.image.source="https://github.com/renovatebot/renovate" \
+      org.opencontainers.image.url="https://renovatebot.com" \
+      org.opencontainers.image.licenses="AGPL-3.0-only" \
+# TODO: org.opencontainers.image.version="$VERSION" \
+      org.opencontainers.image.revision="$SHA"
+
+USER root
+WORKDIR /usr/src/app/
+
+# Build image
+#============
+FROM base as tsbuild
+
+# Python 3 and make are required to build node-re2
+RUN apt-get update && apt-get install -y python3-minimal build-essential
```

Comment:
Adding a `rm -rf /var/lib/apt/lists/*` to this layer should help

Reply:
This is only for build time, not in final layer. but will add it

Reply:
Wouldn't the rm -f then make no difference to built size but slow down the build when executing?

Reply:
@viceice Ah, sorry missed this was `tsbuild` layer.

@rarkins I don't think there will be much of a built time increase. It would help with final build size, but as this is an intermediate image that we'll throw away @viceice is right that it doesn't matter much.

Reply:
Curious, how would it help with final build size? I was referring to the images we actually keep/publish, not the size of the tsbuild

Reply:
It removes the apt cache so it's not included in the layer, therefore it's smaller. There's some more info on [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/):

> In addition, when you clean up the apt cache by removing /var/lib/apt/lists it reduces the image size, since the apt cache is not stored in a layer. Since the RUN statement starts with apt-get update, the package cache is always refreshed prior to apt-get install.